.github/workflows/publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Flexible polyline NPM Publish Workflow
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy-flex-polyline-to-npm:
+    name: Flexible Polyline NPM Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20.x'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Publish to npm
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: |
+          cd javascript
+          npm publish --access public
```

Comment:
Do we need `--access public`? The `package.json` file already specifies that.

Reply:
Right, removed the parameter